ui/src/app/pages/analysis/interactive-notebook.tsx

```diff
@@ -362,52 +355,6 @@ export const InteractiveNotebook = fp.flow(
       }
     }
 
-    private renderPreviewContents() {
```

Comment:
Linting appears to have moved this to the bottom of the file

Reply:
However, there are changes in this function between lines 553 and lines 563.

Reply:
I think your second comment is referring to the other function that was moved.

Reply:
I think that you are right. Thanks for catching that.